app/src/main/java/com/pileproject/drive/programming/visual/activity/BlockListActivity.java

```diff
@@ -57,7 +57,11 @@
 
     public static final String KEY_BLOCK_NAME = "block-list-activity-key-block-name";
 
-    private static final int[] CATEGORY_STRINGS = {R.string.sequence, R.string.repetition, R.string.selection};
+    private static final int[] CATEGORY_STRINGS = {
```

Comment:
This is not a just formatting fix. I removed these `R.string.foo` because they are duplicated and can be replaced with `R.string.programming_foo`. Because They are longer than the previous ones, the line became too long to keep it in one line.